components/dashboard/src/prebuilds/detail/PrebuildTaskTab.tsx

```diff
@@ -54,11 +41,9 @@ export const PrebuildTaskTab = memo(({ taskId, prebuild }: Props) => {
             setActiveToasts((prev) => new Set(prev).add(toastId));
         };
 
-        logEmitter.on("error", errorListener);
```

Comment:
Because we're removing the remaining listener for this event, we can also get rid of it in the `LogEventTypes` inside `prebuild-logs-emitter.ts`.

Reply:
BTW, good catch, it seems we never actually used it anywhere so good it's going away.

Reply:
I think it would be cool to use it for "real" errors, but I'm not sure how to properly test for it. Hence I left it in for future us. :wink: